website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -0,0 +1,353 @@
+---
+description: Deploy Appsmith on a Kubernetes cluster
+sidebar_position: 3
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Kubernetes
+
+[Kubernetes[K8s]](https://kubernetes.io/docs/concepts/overview/) is an open source platform for managing, scaling, and automating the deployment of [containerized applications](https://en.wikipedia.org/wiki/Containerization). It's one of the most popular container orchestration tools. The guide helps you to install [Appsmith](https://github.com/appsmithorg/appsmith) on a [Kubernetes](https://github.com/appsmithorg/appsmith/blob/release/deploy/helm/kubernetes.io) cluster using [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+Before installing Appsmith, a few prerequisites need to be installed and configured.
```

Comment:
in both the points lets just hyperlink once. 

For eg: I see that Helm Package Manager and Helm documentation are the same links.  so lets just hyperlink the main word here

Reply:
Got it.

Reply:
Done.